packages/website/src/bin/page-generator/index.js

```diff
@@ -315,22 +314,20 @@ const generateProjectDocsPages = (docsInfo, generatorConfig, name, urlPath) => {
  * @param pagesPath the absolute path to the `/pages` directory in next
  * @param docsList list of documents
  */
-const cleanPages = (pagesPath, docsList) => {
+const cleanPages = async pagesPath => {
   // This error handling should likely be lifted to when we start executing the whole thing
   if (typeof pagesPath !== 'string' || pagesPath.length < 1) {
     throw new Error(
       "We were worried we were going to erase files from the wrong place so we're stopping",
     );
   }
-  const oldPackagePages = path.join(pagesPath, 'packages');
-  const oldRootPage = path.join(pagesPath, 'readme.js');
-  rimraf.sync(oldPackagePages);
-  rimraf.sync(oldRootPage);
-
-  docsList.forEach(oldDoc => {
-    const oldDocsPaths = path.join(pagesPath, oldDoc.name.toLowerCase());
-    rimraf.sync(oldDocsPaths);
-  });
+
+  await fs.remove(pagesPath);
+};
+
+const addBasePages = async (packageRoot, pagesPath) => {
+  const defaultPagesPath = path.join(packageRoot, 'default-pages');
+  await fs.copy(defaultPagesPath, pagesPath);
```

Comment:
Can we add some tests surrounding this functionality? Do we have any existing tests that test the presence of the default pages?

Reply:
Every test in generatePages fails if this doesn't work.

Reply:
Right, that's not equivalent to testing this functionality though. Is the fail criteria the fact that the 'pages' dir doesn't exist? If so, changing the code to `fs.mkdir('pages')` wouldn't break the other tests but would break functionality.

I think adding a test called 'copies default pages over to pages' would be good for documentation as well.

Reply:
Perhaps a new test that specifically asserts the new behaviour would be good as well.

Reply:
Added a test